seonshine_frontend/src/config/axios.ts

```diff
@@ -1,12 +1,12 @@
 import axios from 'axios';
 
+import { FORBIDDEN, UNAUTHORIZED } from '@/constants/http';
 import { useAuth } from '@/hooks/useAuth';
 import { paths } from '@/routes/paths';
 import { getAccessToken } from '@/utils/persistCache/token';
-import { FORBIDDEN, UNAUTHORIZED } from '@/constants/http';
 
 const axiosInstance = axios.create({
-  baseURL: 'https://dummyjson.com',
+  baseURL: 'http://localhost:5050/',
```

Comment:
please define **env** file for using different environments (develop, production) with name `VITE_API_URL = http://localhost:5050`